README.md

```diff
@@ -43,8 +43,11 @@ This project is early and there's plenty of interesting and challenging work to
 If you find a bug or want to fix a bug, we :heart: PRs and issues! If you see an issue
 in the [queue](https://github.com/gomods/athens/issues) that you'd like to work on, please just post a comment saying that you want to work on it. Something like "I want to work on this" is fine.
 
-If you decide to contribute (we hope you do :smile:), the process is familiar and easy if you've used Github before.
-There are no long documents to read or complex setup.
+If you decide to contribute (we hope you do :smile:), the process is familiar and easy if you've used Github before. There are no long documents to read or complex setup.
```

Comment:
can you attach brians video for first PR: https://www.youtube.com/watch?v=bgSDcTyysRc